controllers/manila_controller.go

```diff
@@ -1142,16 +1224,20 @@ func (r *ManilaReconciler) shareDeploymentCreateOrUpdate(
 	return deployment, op, err
 }
 
-func (r *ManilaReconciler) transportURLCreateOrUpdate(ctx context.Context, instance *manilav1beta1.Manila) (*rabbitmqv1.TransportURL, controllerutil.OperationResult, error) {
+func (r *ManilaReconciler) transportURLCreateOrUpdate(
+	ctx context.Context,
+	instance *manilav1beta1.Manila,
+	rabbitMqClusterName string,
+) (*rabbitmqv1.TransportURL, controllerutil.OperationResult, error) {
 	transportURL := &rabbitmqv1.TransportURL{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%s-manila-transport", instance.Name),
+			Name:      fmt.Sprintf("%s-manila-transport-%s", instance.Name, rabbitMqClusterName),
```

Comment:
For an env update, this will result in having a new transporturl, but still the old with without the suffix. we probably have to either cleanup the old one, or pass an empty string for the RabbitMqClusterName so we only use a suffix for a dedicated NotificationBusInstance.

Reply:
mm  yeah, I didn't think about that: I might need an additional update to this patch to ensure we handle that case. Let me think a bit more about this and I'll come up w/ an update soon.

Reply:
Thank you, I modified the `transportURLCreateOrUpdate` function to keep backward compatibility. As you suggested, passing an empty string results in the original format that we keep for regular `messagingBus` requests.
In case of `notificationBusInstance` (and actually only when we're using a dedicated rabbit), we request a new `transportURL` and we pass the rabbit instance name, used as a suffix.

tl;dr: we're able to keep the same rabbitmq instance w/o modifying it and as per my local tests it works as expected in both greenfield and brownfield scenarios.